docs/source/reference/cli.rst

```diff
@@ -5,6 +5,9 @@ optuna.cli
 
 The :mod:`~optuna.cli` module implements Optuna's command-line functionality using the `cliff framework <https://docs.openstack.org/cliff/latest/index.html>`_.
 
+.. seealso::
+   Examples are provided in the :ref:`cli` tutorial.
```

Comment:
I guess the tutorial page does not show an example but also shows use cases. How about rephrasing it, for example, as follows?
```suggestion
    The :ref:`cli` tutorial provides use-cases with examples.
```